commons/src/methods/convertDateTimeZone.ts

```diff
@@ -0,0 +1,43 @@
+import { DateTime } from "luxon";
+
+/**
+ *
+ * @param slot Date
+ * @param zone string
+ * @returns Date | string
+ */
+export function formatTimeSlot(slot: Date, zone: string): Date | string {
+  try {
+    const dateTimeSlot: DateTime = DateTime.fromJSDate(slot, { zone });
```

Comment:
Does `luxon` have its functions typed? If `fromJSDate` is typed to return a `DateTime`, we shouldn't need to define the type of `dateTimeSlot`.
```suggestion
    const dateTimeSlot = DateTime.fromJSDate(slot, { zone });
```